audit/src/main/resources/config/liquibase/changelog/20190305-create-audit-record-changelog.xml

```diff
@@ -37,23 +40,123 @@
         </createTable>
     </changeSet>
     <changeSet author="tom (generated)" id="1551802246513-2">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="mysql"/>
+        </preConditions>
         <createIndex indexName="created_on_index" tableName="audit_record">
             <column name="created_on"/>
         </createIndex>
     </changeSet>
     <changeSet author="tom (generated)" id="1551802246513-3">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="mysql"/>
+        </preConditions>
         <createIndex indexName="occurred_on_index" tableName="audit_record">
             <column name="occurred_on"/>
         </createIndex>
     </changeSet>
     <changeSet author="tom (generated)" id="1551802246513-4">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="mysql"/>
+        </preConditions>
         <createIndex indexName="origin_index" tableName="audit_record">
             <column name="origin"/>
         </createIndex>
     </changeSet>
     <changeSet author="tom (generated)" id="1551802246513-5">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="mysql"/>
+        </preConditions>
         <createIndex indexName="user_index" tableName="audit_record">
             <column name="user"/>
         </createIndex>
     </changeSet>
+
+    <changeSet author="Ivo Zaal" id="1551802246513-6">
```

Comment:
These should run without issues, but I do want to point out we should try to never add new changesets to existing changelogs for already supported DBMSs. These should be in a separate changelog.

Reply:
No sure. i feel like it would be more clear for future development if we group the changesets for the same purpose together.

Reply:
Isn't that the opposite of what you want? By adding changesets to an existing changelog, you create a difference in how changesets are executed for already existing projects with up and running production environments vs new projects. I would group it based on release. For the postgres changesets, these obviously have to happen when the original mysql changesets were planned, but anything else should be in a 9.3.0 or 9.4.0 changelog, or whatever version the changes are for.

Reply:
These changesets are meant to make sure that the mysql and postgres index names are the same though. If we put them in a new file, where are the postgres changesets meant to go? they are not completely equivalent

Reply:
The postgres changes are all necessary, so I would leave them here and only move the mysql changes instead.

Reply:
I'm still not sure i completely agree, but i moved the changesets to a new changelog